src/Lib/Lionk.Core/Model/Component/Measure.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright © 2024 Lionk Project
+
+namespace Lionk.Core.Component;
+
+/// <summary>
+/// This class represents a measure.
+/// </summary>
+/// <typeparam name="T"> The type of the value. </typeparam>
+public class Measure<T>
+{
+    private T? _value;
+
+    /// <summary>
+    /// Gets the name of the measure.
+    /// </summary>
+    public string MeasureName { get; private set; }
+
+    /// <summary>
+    /// Gets the time of the measure.
+    /// </summary>
+    public DateTime? Time { get; private set; }
+
+    /// <summary>
+    /// Gets the unit of the value.
+    /// </summary>
+    public string Unit { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the value of the measure.
+    /// </summary>
+    public T? Value
```

Comment:
I don't think that we need to create one measure and set is value each time a new value is available.

It think that a new measure must be created each time we do a new measure. So it's preferable to only have private set and set all this data from ctor.

Reply:
I think that because, if we want to have an history or something like that, we need a new Measure foreach measure done.

Reply:
I don't think we're talking about the same type of structure for the `List<Measure>`, I use this list to store the various measurements of a sensor. Let's imagine a GPS, it will store a position (x, y, z, which will probably be a `Position` object, and maybe a list of satellites that have given this position.

As a result, my list of measures will include:
```C#
public List<Measure<object>> Measures { get; } = new()
{
    new Measure<Position>("Position", new () ),
    new Measure<List<Satelites>>("Satelites",new() ),
}
```
I think the List can be a simple Measure[]

For an object that needs to keep a history, an `IMeasureHistory` must be implemented.

To resolve this question without `IMeasureHistory`

We can create a GpsMeasure object to make a simple .Add in `Measures<GpsMeasure>`.